standalone-metastore/metastore-common/src/main/thrift/hive_metastore.thrift

```diff
@@ -1696,13 +1707,28 @@ struct ExtendedTableInfo {
  4: optional list<string> requiredWriteCapabilities // capabilities required for write access
 }
 
+struct DropTableRequest {
+ 1: required string dbName,
+ 2: required string tableName,
+ 3: optional string catalogName,
```

Comment:
same here. It could make sense for catalogName to be the first.

Reply:
Ack